src/server/components/dashboard/annualReview/controllers.ts

```diff
@@ -79,7 +80,13 @@ async function updateNewAnnualReviewDate(req: Request, res: Response): Promise<v
   const annualReviewDate = new Date(newAnnualReviewDate as string);
   const newAnnualReviewDateFormatted = DateFns.format(annualReviewDate, DATE_FORMAT);
 
-  await updateAnnualReviewDate(listId, annualReviewDate.toISOString());
+  try {
+    await updateAnnualReviewWithKeyDates(list, annualReviewDate.toISOString());
```

Comment:
probably needs discussion with the wider team but the updating of the key dates may need locking down after annual review starts.  Can be treated as a separate PR though.